CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Changelog
 
+### AMI Release v20211117
+* amazon-eks-gpu-node-1.21-v20211117
+* amazon-eks-gpu-node-1.20-v20211117
+* amazon-eks-gpu-node-1.19-v20211117
+* amazon-eks-gpu-node-1.18-v20211117
+* amazon-eks-gpu-node-1.17-v20211117
+* amazon-eks-arm64-node-1.21-v20211117
+* amazon-eks-arm64-node-1.20-v20211117
+* amazon-eks-arm64-node-1.19-v20211117
+* amazon-eks-arm64-node-1.18-v20211117
+* amazon-eks-arm64-node-1.17-v20211117
+* amazon-eks-node-1.21-v20211117
+* amazon-eks-node-1.20-v20211117
+* amazon-eks-node-1.19-v20211117
+* amazon-eks-node-1.18-v20211117
+* amazon-eks-node-1.17-v20211117
+
+Binaries used to build these AMIs are published:
+* s3://amazon-eks/1.21.5/2021-11-10/
+* s3://amazon-eks/1.20.11/2021-11-10/
+* s3://amazon-eks/1.19.15/2021-11-10/
+* s3://amazon-eks/1.18.20/2020-09-02/
+* s3://amazon-eks/1.17.17/2020-09-02/
+
+AMI details:
+* kernel: 5.4.156-83.273.amzn2 (1.19 and above), 4.14.252-195.483.amzn2 (1.18 and below)
+* dockerd: 20.10.7-5.amzn2
+* containerd: 1.4.6-7.amzn2
+* runc: 1.0.0-2.amzn2
+* cuda: 470.57.02
+* nvidia-container-runtime-hook: 1.4.0-1.amzn2
+* SSM agent: 3.0.1124.0-1.amzn2
+
+Notable changes:
+Update `containerd` to `1.4.6-7.amzn2` and `docker` to `20.10.7-5.amzn2` to patch vulnerabilities in [CVE-2021-41190](https://alas.aws.amazon.com/ALAS-2021-1551.html)
```

Comment:
I could be wrong, but I think this part of the changelog is incorrect. This update is not included in AMI Release v20211117. The fix for that cve came in [this commit](https://github.com/awslabs/amazon-eks-ami/commit/b209f51c82174ee9c3eee90a714276378f390db4) which was committed after [the sha currently linked with v20211117](https://github.com/awslabs/amazon-eks-ami/commit/216a53b7650313ec02e06bb42e9e4efc329396f6)

Reply:
I agree the changelog is incorrect